README.md

```diff
@@ -30,6 +30,40 @@ The context is configured through `delta-context-config.js` which is covered in
 
 The `DCR_LANDING_ZONE_GRAPH` is maintained by the delta-producer when this feature is and contains all the triples from the data-sources producer graph - without any filtering or other changes. This graph is used to lookup context and can be offloaded to a different triplestore than the main application database by providing the `DCR_LANDING_ZONE_DATABASE` environment variable.
 
+### Delta Message - SPARQL mapping
+
+The delta-consumer provides a way to map the incoming messages to a different model. This is done by providing a SPARQL queries in the configuration directory. The queries are executed on the landing zone graph and the results are used to update target graph.
+
+Each triple from the delta message is treated individually.
+
+#### Delete
+
+When a delete comes in, which breaks the where part of a query - the whole construct clause is deleted from the target graph.
```

Comment:
I remember we discovered some kind of 'unexpected' behavior; when the following mapping would be done:

Suppose incoming delta:
```
DELETE <subject> skos:inScheme <http://foo>
```

and mapping:
```
CONSTRUCT {
   ?s skos:prefLabel ?label.
} WHERE {
   ?s skos:inScheme <http://foo>;
     skos:prefLabel ?label.
}
```

would result in:
```
DELETE <subject> skos:prefLabel "label"
```

which might feel unexpected; but I remember we talked a fair bit about it; and I think we ended with what has been described in the `README.md`. But I can't find it in my notes, so maybe, is that what I am looking for: a wrap-up of the discussion about these 'unexpected' deltas? Do you have it somewhere?